upstairs/src/live_repair.rs

```diff
@@ -4179,7 +4103,7 @@ pub mod repair_test {
         // same extent.
         assert_eq!(
             jobs[3].work.deps(),
-            &[jobs[2].ds_id, jobs[1].ds_id, jobs[0].ds_id]
+            &[jobs[0].ds_id, jobs[1].ds_id, jobs[2].ds_id]
```

Comment:
Let's use the `hashset` function here to avoid tests failing if the order changes

Reply:
Dependencies should always be ordered after this PR goes in, since they're stored in a `BTreeSet` during collection.  This means we could get rid of `hashset` entirely, if we want to implicitly test this behavior; alternatively, we could keep using it for future flexibility.

Reply:
Oh, I apparently _already_ removed the `hashset` function, so... mission accomplished?